src/wepy/resampling/novelty/weight_novelty.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+from wepy.rsampling.novelty.novelty import Novelty
+
+class Weight_Factor_Novelty(Novelty):
+
+    def __init__(self, n_walkers, pmin=1e-12):
+        
```

Comment:
If you want to use the constructor for the super class you should call:

```
super().__init__(n_walkers, pmin=pmin,...)
```

I'm guessing this is why you set them as class variables above.

Reply:
Removed the superclass dependency for the Novelty object and removed the class variables.